docs/superpowers/specs/2026-05-11-comet-style-url-title-ux-design.md

```
@@ -0,0 +1,226 @@
+  /** Page title from last navigation */
+  pageTitle: string | null;
+  /** Favicon URL from page-favicon-updated */
+  faviconUrl: string | null;
+  /** Called when user submits a URL (Enter or Go) */
+  onNavigate: (url: string) => void;
+}
+```
+
+### Component tree
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)